### PR TITLE
#ATOR-283 Changed columns, added tooltips, buttons used from library

### PR DESCRIPTION
**Pull request by EarlOld**

### CODE REVIEW

File: stores/useUserStore.ts [hunk 1]
```diff
@@ -78,6 +79,7 @@ export const useUserStore = defineStore('user', {
 
       if (claimable.status === 200) {
         const relays = await claimable.json();
+        debugger;
```
Comment (yumirkov) [inline stores/useUserStore.ts:82]: what is it?

File: components/DataTableMyRelays.vue [hunk 2]
```diff
@@ -64,15 +66,20 @@ const {
...
-          consensusWeight: item.relay.consensus_weight,
-          observedBandwidth: item.relay.observed_bandwidth,
-        };
-      });
+      const structuredResults = trimmedResults.reduce(
+        (acc, item) => {
+          acc[item.relay.fingerprint] = {
+            nickname: item.relay.nickname || '-',
+            active: item.relay.running,
+            status: 'verified',
```
Comment (yumirkov) [inline components/DataTableMyRelays.vue:74]: is it okay to hardcode it here?

Comment (EarlOld) [reply components/DataTableMyRelays.vue]: Fixed

File: stores/useUserStore.ts (hunk 1 above)
Comment (EarlOld) [reply stores/useUserStore.ts]: deleted

**Review by floodcode — APPROVED** [review]

**Review by yumirkov — APPROVED** [review]

**Pull request merged**